Delicious.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+  <html>
+      <head>
+      <title>Delicous Blog</title>
+      <link href="css/style.css" type="text/css" rel="stylesheet" />
+      <link href='http://fonts.googleapis.com/css?family=Yellowtail' rel='stylesheet' type='text/css'>
+    </head>
+   
+    <body>
+      <header>
+      		<img src = "img/logo.png" />
```

Comment:
Do not add spaces in between your attributes and values. It should be like this:

```
attribute="value"
```